photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -160,6 +171,28 @@ final class Driver(val sc: SparkContext, val params: GameParams, implicit val lo
     }
 
   /**
+   * Run various checks on data (training or validation. Throws a RuntimeException if any of the checks does not pass.
+   *
+   * @note this is somewhat expensive, so we can turn it off
+   *
+   * Checks performed
+   * ================
+   * - all weights >= 0.0 and at least 1 weight > 0
+   *
+   * @param data The data to check
+   */
+  protected[training] def checkData(data: DataFrame): DataFrame =
+
+    if (params.checkData)
+      data
+        .filter(col("weight") >= 0.0)
+        .tap { df => if (df.count == 0)
```

Comment:
Could you please move the start of the `if` statement to the next line? Think of it like your method header and body separation style preference